### PR TITLE
Recursive Copy Fix & MSdeploy Changes

### PR DESCRIPTION
**Pull request by vincentdass**

### CODE REVIEW

File: Tasks/Common/webdeployment-common/utility.ts [hunk 1]
```diff
@@ -153,4 +153,25 @@ export async function isMSDeployPackage(webAppPackage: string ) {
     }
     tl.debug("Is the package an msdeploy package : " + isParamFilePresent);
     return isParamFilePresent;
+}
+
+export function copyDirectory(sourceDirectory: string, destDirectory: string) {
```
Comment (kmkumaran) [inline Tasks/Common/webdeployment-common/utility.ts]: As discussed, can you add L0 for this.

Comment (vincentdass) [inline Tasks/Common/webdeployment-common/utility.ts]: L0 Test added.

File: Tasks/AzureRmWebAppDeployment/Tests/L0.ts [hunk 2]
```diff
@@ -458,4 +458,13 @@ describe('AzureRmWebAppDeployment Suite', function() {
         done();
     });
 
+     it('Validate webdeployment-common.utility.copyDirectory()', (done:MochaDone) => {
```
Comment (arjgupta) [inline Tasks/AzureRmWebAppDeployment/Tests/L0.ts]: please add the same test in IISWebAppDeploymentOnMachineGroup task as well

Comment (vincentdass) [inline Tasks/AzureRmWebAppDeployment/Tests/L0.ts]: added L0 Test for IIS App Deploy

**Pull request merged**